src/loaders/common/loader_helper.py

```diff
@@ -485,3 +494,49 @@ def _get_help_string(self, action):
         if action.default is None or action.default is False:
             return action.help
         return super()._get_help_string(action)
+
+
+def transform_heatmap_row_cells(data: dict):
+    """
+    Transform the cells structure in a heatmap row to a new structure.
+
+    The new structure is a set of keys and values where the keys are constructed from the old structure.
+    new structured key format: <HEATMAP_COL_PREFIX>_<col_id>_<FIELD_HEATMAP_CELL_ID|FIELD_HEATMAP_CELL_VALUEl>
+
+    e.g.
+
+    The old structure:
+    "cells": [
+        {
+            "cell_id": "cell_0",
+            "col_id": "0",
+            "val": 0.0
+        },
+        {
+            "cell_id": "cell_1",
+            "col_id": "1",
+            "val": 1.0
+        }
+    ]
+
+    The new structure:
+    "col_0_cell_id": "cell_0",
+    "col_0_val": 0.0,
+    "col_1_cell_id": "cell_1",
+    "col_1_val": 1.0
+
+    """
+
+    # Iterate over the 'cells' structure and remove them from the data structure while constructing the new structure
+    for cell in data.pop(FIELD_HEATMAP_ROW_CELLS):
```

Comment:
It does seem a little weird to do the transform here, but I suppose we can push it back to the tools if it becomes a problem.

Reply:
yea. Other than that, I think it will be easier for us to revert/skip this step if Arango nested search did become available in Community Edition.